app/src/main/java/com/nervousfish/nervousfish/modules/cryptography/KeyGeneratorAdapter.java

```diff
@@ -28,4 +36,41 @@ private KeyGeneratorAdapter(final IServiceLocatorCreator serviceLocatorCreator)
     public static ModuleWrapper<KeyGeneratorAdapter> newInstance(final IServiceLocatorCreator serviceLocatorCreator) {
         return new ModuleWrapper<>(new KeyGeneratorAdapter(serviceLocatorCreator));
     }
+
+    /**
+     * Generates a random KeyPair with the RSA algorithm.
+     * @return a randomly generated KeyPair
+     */
+    public static KeyPair generateRandomKeyPair() throws NoSuchAlgorithmException {
```

Comment:
I don't really like this method name... I would suggest changing it to `generateRSAKeyPair()`, removing the random key pair as that is not really relevant and adding the key pair type so you don't have to go and read the documentation in order to know what key you're generating.

Having said that, I would update the documentation to something like: 
```java
/**
 * Generate a RSA {@link KeyPair}.
 *
 * @return A randomly generated {@link KeyPair}
 */
```